### PR TITLE
LPS-189274 [Marketplace] Fix App Marketplace Dashboard Style Adjustments

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -1,87 +1,98 @@
...
 
-	@media screen and (min-width:600px) and (max-width: 899px) {
-		.adt-solutions-search-results .cards-container {
-			grid-template-columns: repeat(2, minmax(0, 1fr));
-		}
+@media screen and (min-width: 600px) and (max-width: 899px) {
+	.adt-solutions-search-results .cards-container {
+		grid-template-columns: repeat(2, minmax(0, 1fr));
 	}
+}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:35]: It needs to be formatted.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 2]
```diff
@@ -1,87 +1,98 @@
...
+						</#if>
+					</#list>
+				</#if>
+
 				<#list images as image>
 						<#assign title = image.getTitle()!"" />
 						<#if title?contains("App Icon")>
 							<#assign productImageURL = image.getURL() />
 						</#if>
 				</#list>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:90]: It needs to be formatted.

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]

**Comment by joanagrego** [comment]
Just starting reviewing :)

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/pt-liferay-solutions/liferay-portal/pull/4485)

**Pull request closed without merging**